fastai/callbacks/mlflow.py

```diff
@@ -0,0 +1,44 @@
+"A `Callback` that saves tracked metrics and notebook file into MLflow server."
+from ..torch_core import *
+from ..callback import *
+from ..basic_train import Learner, LearnerCallback
+#This is an optional dependency in fastai.  Must install separately.
+try: import mlflow
+except: pass
```

Comment:
Maybe add an error here that would give the instructions on how to install mlflow? Silently failing isn't a beahvior we would want.